docs/guides/2-cli.md

```diff
@@ -60,6 +61,86 @@ Here you can build a [custom ruleset](../getting-started/3-rulesets.md), or exte
 - [OpenAPI ruleset](../reference/openapi-rules.md)
 - [AsyncAPI ruleset](../reference/asyncapi-rules.md)
 
+## Scoring the API
+
+Scoring an API definition is a way to understand in a high level, how compliant is the API definition with the rulesets provided. This helps teams to understand the quality of the APIs regarding the definition.
```

Comment:
Scoring an API definition is a way to understand in a high level, how compliant is the API definition with the rulesets provided.

Suggest

Scoring an API definition is a way to understand at a high level how compliant the API definition is with the rulesets provided.